gptscript_test.go

```diff
@@ -160,7 +160,7 @@ func TestCancelRun(t *testing.T) {
 }
 
 func TestAbortChatCompletionRun(t *testing.T) {
-	tool := ToolDef{Instructions: "What is the capital of the united states?"}
+	tool := ToolDef{Instructions: "Generate a real long essay about the meaning of life."}
```

Comment:
I have to fix this. It is flaky and will fail the test if the message are sent in one event